js/main.js

```diff
@@ -288,26 +336,51 @@ const onActiveMode = () => {
 
 toggleDisabledOnFormNodes();
 
-mapPinMain.addEventListener(`mousedown`, function (evt) {
-  if (evt.button === 0) {
-    onActiveMode();
-    initPinsScreen();
-    passAddressInput();
-  }
-}, {
-  once: true
-});
+let cardNode;
+
+const closeCard = () => {
+  cardNode.parentNode.removeChild(cardNode);
+  document.removeEventListener(`keydown`, onPopupEscPress);
+};
 
-mapPinMain.addEventListener(`keydown`, function (evt) {
-  if (evt.key === KeyboardKeys.ENTER) {
-    onActiveMode();
-    initPinsScreen();
-    passAddressInput();
+const onPopupEscPress = function (evt) {
+  if (evt.key === KeyboardKeys.ESCAPE) {
+    evt.preventDefault();
+    closeCard();
   }
+};
+
+mapPinMain.addEventListener(`click`, function () {
+  onActiveMode();
+  initPinsScreen();
+  passAddressInput();
+  getPins();
+
+  let pinsArr = Array.from(mapPins);
+  pinsArr.shift();
```

Comment:
Зачем делать шифт? просто не выбирай лишнего
```suggestion
    let pinsArr = Array.from(mapPinsNode.querySelectorAll(`.map__pin:not(.map__pin--main)`));
```
Предлагаю избавиться от этого:
![image](https://user-images.githubusercontent.com/30810655/95363689-74c5a480-08d8-11eb-95d1-671a77315ffb.png)
Неясно зачем это